style.css

```diff
@@ -489,6 +433,94 @@ li {
   border: solid 1px #5e6c84;
 }
 
+.imgSrc {
+  width: 20px;
+  height: 20px;
+}
+
+.work-section-popup.close-popup {
+  display: none;
+}
+
+.work-wrapper {
+  background: rgb(193, 199, 208, 0.7);
+  backdrop-filter: blur(8px);
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+}
+
+.work-card-pop {
+  background-color: rgb(255, 255, 255);
+  width: 82%;
+  margin: auto auto;
+  border-radius: 1.2rem;
+  min-height: 500px;
+  min-height: 70%;
+  height: 95%;
+}
+
+.work-card-pop-close {
+  font-size: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding: 0 5% 0 7%;
+}
+
+.work-details-pop {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  margin: 0 auto;
+  width: 87%;
+  height: 87%;
+}
+
+.pop-large {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  padding-bottom: 10%;
+}
+
+.pop-large > p {
+  font-size: 13px;
+}
```

Comment:
- [ ] kindly change `font-size` to `16px` to match Figma template.

Reply:
Thanks [omarsalem7](https://github.com/omarsalem7)  for the good eyes.